addons/mail/static/src/models/message/message.js

```diff
@@ -608,6 +616,16 @@ function factory(dependencies) {
             default: false,
             dependencies: ['isModeratedByCurrentPartner'],
         }),
+        /**
+         * Determine whether the message has unfollow button or not
+         */
+        hasUnfollowIcon: attr({
```

Comment:
This name is good for the component prop but it is not good for the model field.
Here I'm suggesting to have explicit name `isCurrentPartnerFollowingOriginThread`.

Reply:
Maybe this field is not even useful actually.

It's fine to use `message.originThread.isCurrentPartnerFollowing` directly in the template.